front/css/login.css

```diff
@@ -0,0 +1,76 @@
+.login_wrapper {
```

Comment:
I'm not a specialist in CSS but it more common to use snake-case in class naming (for blocks).
We should choose a naming convention, basicaly we could try to use the BEM convention. Issue created for it : #103